### PR TITLE
fixed extrapolation bug with bounds_error

### PR DESCRIPTION
**Pull request by aimalz**
The error was caused by using the `fill_value` keyword of `scipy.interpolate.interp1d` without the `bounds_error` keyword.  Problem solved!

### CODE REVIEW

**Comment by aimalz** [comment]
@drphilmarshall, would you mind reviewing this bug fix requested by @sschmidt23? Thanks!

File: qp/pdf.py [hunk 1]
```diff
@@ -322,7 +323,7 @@ def gmm(x, *args):
             stdevs = popt[2]
         else:
             if self.samples is None:
-                self.samples = self.sample(using=using)
+                self.samples = self.sample(using=using, vb=vb)
```
Comment (drphilmarshall) [inline qp/pdf.py:326]: Did you mean to leave the `vb` setting on?

**Review by drphilmarshall — APPROVED** [review]
Looks good - might want to turn vb off before merging, if your tests all pass?

File: qp/pdf.py (hunk 1 above)
Comment (aimalz) [reply qp/pdf.py:326]: I think this change just causes `vb` from the call to `mix_mod_fit` to be propagated down.  It should solve the problem wherein performing the `mix_mod_fit` on a `qp.Ensemble` is significantly slowed due to printing all the samples, even when the call to `mix_mod_fit` has `vb=False`.

**Pull request merged**